front/app/components/SeatInfo/utils.ts

```diff
@@ -0,0 +1,38 @@
+import { TSeatNumber } from 'api/app_configuration/types';
+import { isNil } from 'utils/helperUtils';
+
+type ExceededLimitType = {
+  hasReachedOrIsOverLimit: boolean;
+  hasExceededSetSeats: boolean;
+};
+
+// TODO: Remove / simplify this function after seat based billing
+// Pulling this out to reduce comlexity and make it easy to read where it is used. This will go away once we are stable and have enabled the second iteration of seat based billing for all clients
+export const getExceededLimitInfo = (
+  hasSeatBasedBillingEnabled: boolean,
+  currentSeatTypeSeats: number, // admins_number/project_moderators_number
+  additionalSeatTypeSeats: TSeatNumber, // additional_admins_number/additional_moderators_number
+  maximumSeatTypeSeats: TSeatNumber // maximum_admins_number/maximum_moderators_number
```

Comment:
The / makes it look live it's a fraction since the type is number. Let's replace with 'or'?